src/main/java/org/bytedeco/javacpp/Loader.java

```diff
@@ -126,6 +126,8 @@ public static String getPlatform() {
                 osArch = "x86_64";
             } else if (osArch.startsWith("aarch64") || osArch.startsWith("armv8") || osArch.startsWith("arm64")) {
                 osArch = "arm64";
+            } else if (osArch.startsWith("loongarch64")) {
+                osArch = "loongarch64";
```

Comment:
This doesn't change the value of osArch, so we don't need to add anything here. Please remove this.

Reply:
> This doesn't change the value of osArch, so we don't need to add anything here. Please remove this.

Done

Reply:
I'm assuming osArch is always "loongarch64" here though. Let me know if this isn't the case.

Reply:
> I'm assuming osArch is always "loongarch64" here though. Let me know if this isn't the case.
osArch value is always "loongarch64", you've made the right suggestion.
Thanks

Reply:
> I'm assuming osArch is always "loongarch64" here though. Let me know if this isn't the case.

osArch value is always "loongarch64", you've made the right suggestion.
Thanks